build.sbt

```diff
@@ -1,10 +1,12 @@
+import com.typesafe.sbt.packager.debian.DebianPlugin.autoImport.Debian
+
 name := """amiable"""
 
-version := "1.0-SNAPSHOT"
+version := "1.2-SNAPSHOT"
```

Comment:
Is this change needed?

Reply:
Good spot! Have reverted version back to "1.0-SNAPSHOT"